pineappl/src/grid.rs

```diff
@@ -222,6 +223,51 @@ pub struct Grid {
     more_members: MoreMembers,
 }
 
+/// Data type that indentifies different types of convolutions.
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
Do you left `Debug` intentionally?

Reply:
`Debug` is needed whenever you compare objects using assertions, which is done here:

https://github.com/NNPDF/pineappl/blob/5eba2360f84c503e2a1fc4b1c8c9f49757cd5f93/pineappl/src/grid.rs#L1925-L1951